src/main/java/com/github/thundermarket/thundermarket/aspect/AvailableWithoutSession.java

```diff
@@ -0,0 +1,11 @@
+package com.github.thundermarket.thundermarket.aspect;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface AvailableWithoutSession {
```

Comment:
without도 결국 부정표현이라서요. '세션 없이 접근 가능하다'라는 표현이 너무 구체적인 것 같아요. 로그인 방식이 세션이 아닌 다른 방식으로 바뀔수도 있으니까요. 조금 더 추상화 할 필요가 있어 보입니다. 아래 이름으로 제안 드릴게요.
```suggestion
public @interface Authenticated {
```

Reply:
4990f7a4da09f36568a4ff1baf41727a173fbef8
AOP, 어노테이션 생성이 생소해서 추상적으로 이름 짓기가 잘 안 됐던 것 같아요.
인증된 상태라는 최종 결과에 집중하는 이름으로는 추천해주신 `Authenticated`가 좋은 것 같습니다.
감사합니다~~